Microsoft.DotNet.Interactive.CSharp/ScriptExecutionExtensions.cs

```diff
@@ -25,7 +25,8 @@ internal static class ScriptExecutionExtensions
                                     return (ScriptState<object>) null;
                                 }, cancellationToken));
 
-            return completed.Result;
+            var result = await completed;
```

Comment:
This shouldn't be necessary because the task has already been awaited once.

Were you seeing an issue?

Reply:
I'm exploring to try to figure out why `dotnet test` is hanging in CI and one thread suggested not fully-awaited tasks, but that doesn't seem to be the case.  I'll keep looking.